lib/comgr/src/comgr-compiler.cpp

```diff
@@ -671,6 +671,21 @@ amd_comgr_status_t AMDGPUCompiler::removeTmpDirs() {
   return AMD_COMGR_STATUS_SUCCESS;
 }
 
+amd_comgr_status_t AMDGPUCompiler::executeOutOfProcessHIPCompilation(
+    llvm::ArrayRef<const char *> Args) {
+    StringRef Exec = "/opt/rocm/hip/bin/hipcc";
```

Comment:
I think you should check the env HIP_PATH here ifnot choose this location

Reply:
> I think you should check the env HIP_PATH here ifnot choose this location

fixed